persistence/jdbc/src/main/java/org/infinispan/persistence/jdbc/configuration/TableManipulationConfigurationBuilder.java

```diff
@@ -37,11 +36,11 @@
    }
 
    /**
-    * Repetitive DB operations this are batched according to this parameter. This is an optional parameter, and if it
-    * is not specified it will be defaulted to {@link TableManager#DEFAULT_BATCH_SIZE}.
+    * Deprecated. Please use {@link org.infinispan.configuration.cache.AbstractStoreConfigurationBuilder#maxBatchSize(int)} instead.
     */
+   @Deprecated
    public S batchSize(int batchSize) {
-      attributes.attribute(BATCH_SIZE).set(batchSize);
+      attributes.attribute(AbstractJdbcStoreConfiguration.MAX_BATCH_SIZE).set(batchSize);
```

Comment:
I was actually thinking more along something like `builder.batchSize(batchSize)` by invoking the batchSize method on the `StoreConfiguration` parent. This then requires changing the create method though as well. I guess it isn't really worth changing then maybe.